sdk/core/core-paging/src/getPagedAsyncIterator.ts

```diff
@@ -29,19 +29,19 @@ export function getPagedAsyncIterator<
     byPage:
       pagedResult?.byPage ??
       ((settings?: PageSettings) => {
-        return getPageAsyncIterator(
-          pagedResult as PagedResult<TPage, PageSettings, TLink>,
-          settings?.maxPageSize
-        );
+        const { continuationToken, maxPageSize } = settings ?? {};
```

Comment:
same question here, this seems scoped and easy to review, but harder to have clear signoff when it's part of a much larger PR

Reply:
I duplicated it in https://github.com/Azure/azure-sdk-for-js/pull/21794